### DISCUSSION TITLE
Welcome to gelfand-trigonometry-solutions Discussions!

### DISCUSSION THREAD

**Discussion by philip-healy** [philip-healy/gelfand-trigonometry-solutions#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AsbestosSniffer** [comment]
Hello,

Around two years ago now, I started working on a solutions manual for Gelfand's _Trigonometry_ for ["service" hours](https://en.wikipedia.org/wiki/Creativity,_activity,_service) which were required to graduate high school. I completed only half of the book while omitting many answers that I thought were easy/repetitive, but uploaded it anyway to [archive.org](https://archive.org/details/gelfand-trigonometry-solutions-manual) less than a year ago at the end of high school.

It didn't cross my mind to do a simple Google search or else I surely would have found this project earlier and abandoned mine. I didn't  refer to this project in any way while writing mine, having only found out about it today, and I think a perusal of my .tex files should bear this out (for example I used tkz-euclide to draw diagrams), to say nothing of the high probability of some of my answers being incorrect.

Nevertheless, if you have any doubts about plagiarism, I have no objection to deleting my files from the Internet Archive (assuming it is possible: I don't exactly know how the Internet Archive works) as I've already graduated.

**Reply by philip-healy** [reply]
Hi @H2Oenjoyer,

awesome work! I have updated the project's README and solutions document to link to your solution set. Feel free to create a PR substituting existing solutions for your ones, where you feel that your ones are better. Also, you might be able to fill in the last few missing solutions. For example, there are missing solutions for the exercises on pages 8 and 9.

**Comment by djshuster** [comment]
Hello! I believe there may be an error in the solutions for Chapter 9 p. 213 exercise #3, an error which I think Gelfand may have made as well (wow I am bold to say this. In fact, I think the error goes back to the "summary" at the bottom of page 212.) But anyway, it seems to me that arccos(b) will always represent a value between 0 and pi, which means that sin (arccos(b)) will always be positive.

Similarly, for exercise 4, we actually can just make our solution $\frac{b}{\sqrt{1-b^2}}$. When b is positive, arcsin will be positive (1st quadrant angle), so tan will be positive, as our solution is.
When b is negative, arcsin will be negative (4th quadrant angle), so tan will be negative, as our solution would be with negative b.

Please do correct me if I'm wrong. I'm happy to type these up in LaTeX but overleaf is taking a long time to compile things and I don't wanna ruin any nice work before we all agree on this 😅

**Reply by philip-healy** [reply]
You are correct about the range of arccos: https://www.math.tamu.edu/~austin/section4_6.pdf

So I do think that your reasoning is sound. @sameeMushtak do you agree?

**Reply by sameeMushtak** [reply]
Yes, I agree with @djshuster's analysis for these 2 exercises. I graphed y = sin(arccos x) in Desmos just to confirm that the function is indeed non-negative on [-1,1].

**Reply by sameeMushtak** [reply]
@philip-healy I'm noticing that there are also some errors in exercise 2 of p. 213. It might be a good idea to do a review of the solutions we have for Chapter 9.
